### PR TITLE
Removing demux file

### PR DESCRIPTION
**Pull request by josenavas**
Removing demux file, updating travis and updating the installation instructions

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -27,6 +27,7 @@ install:
     'pandas>=0.18' 'matplotlib>=1.1.0' 'scipy>0.13.0' 'numpy>=1.7' 'h5py>=2.3.1'
   - source activate env_name
   - pip install sphinx sphinx-bootstrap-theme coveralls 'ipython[all]==2.4.1'
+  - pip install https://github.com/qiita-spots/qiita-files/archive/master.zip --no-deps
```
Comment (antgonza) [inline .travis.yml]: Why --no-deps and shouldn't this be automatically installed (added to setup.py)?

**Comment by ElDeveloper** [comment]
👍

**Comment by josenavas** [comment]
I'm wondering if coveralls decreased because I removed a I file that had good coverage? Or is this a result of the coveralls change @antgonza ?

File: INSTALL.md [hunk 2]
```diff
@@ -109,6 +109,11 @@ Install the development version of moi:
 pip install https://github.com/biocore/mustached-octo-ironman/archive/master.zip --no-deps
 ```
 
+Install the development version of qiita-files:
+```bash
```
Comment (antgonza) [inline INSTALL.md]: This is not needed any more, right?

**Comment by antgonza** [comment]
Looking at the logs it seems like those are changes based on how the new coveralls works.

File: INSTALL.md (hunk 2 above)
Comment (josenavas) [inline INSTALL.md]: Right

**Comment by josenavas** [comment]
boom! Thanks for the tips about how to fix coverage @antgonza !!

**Pull request merged**